testing/integration/beats_serverless_test.go

```diff
@@ -0,0 +1,589 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+//go:build integration
```

Comment:
Having just the `integration` build tag will include this file while trying to launch integration tests for elastic agent. Maybe we want to specify an additional tag like:

```suggestion
//go:build integration
//go:build serverless
```
Ofc we need to add the tag in magefile target

Reply:
Yah, considering that this PR is technically temporary until we get buildkite setup in the beats repo, I didn't want to get to clever with anything that would require modifications to the agent code, since eventually this'll become a proper integration test in the agent repo.

Reply:
Changing the build tag wouldn't just require you to make a dedicated mage target for running these I think. That is probably a good thing.

How are these getting excluded from `mage integration:test` runs today?